### PR TITLE
Interactive content PR for questions and comments

### PR DESCRIPTION
**Pull request by henkbirkholz**

### CODE REVIEW

File: draft-ietf-scitt-architecture.md [hunk 1]
```diff
@@ -123,14 +123,12 @@ informative:
 
 --- abstract
 
-Traceability of physical and digital Artifacts in supply chains is a long-standing, but increasingly serious security concern.
+Traceability of physical and digital artifacts in supply chains is a long-standing, but increasingly serious security concern.
```
Comment (SteveLasker) [inline draft-ietf-scitt-architecture.md:126]: Artifact is defined in terminology, which we've said we want to keep Capatilized.

File: draft-ietf-scitt-architecture.md [hunk 2]
```diff
@@ -123,14 +123,12 @@ informative:
...
 
 This document defines a generic, interoperable and scalable architecture to enable transparency across any supply chain with minimum adoption barriers.
-It provides flexibility, enabling interoperability across different implementations of Transparency Services with various auditing and compliance requirements.
-Issuers can register their Signed Statements on any Transparency Service, with the guarantee that all Consumers will be able to verify them.
-
-Within the SCITT Architecture, a producer is known as an Issuer, and a consumer is known as a Verifier.
+It provides flexibility, enabling interoperability across different implementations of transparency services with various auditing and compliance requirements.
+Issuers can register their signed statements on any transparency service, with the guarantee that all consumers will be able to verify them.
```
Comment (SteveLasker) [inline draft-ietf-scitt-architecture.md:131]: Same note about capitalizing termonilogy. 
https://ietf-wg-scitt.github.io/draft-ietf-scitt-architecture/draft-ietf-scitt-architecture.html#name-terminology
> When used in text, the corresponding terms are capitalized.

File: draft-ietf-scitt-architecture.md [hunk 3]
```diff
@@ -139,9 +137,9 @@ Within the SCITT Architecture, a producer is known as an Issuer, and a consumer
 This document describes a scalable and flexible, decentralized architecture to enhance auditability and accountability across various existing and emerging supply chains.
 It achieves this goal by enforcing the following complementary security guarantees:
 
-1. Statements made by Issuers about supply chain Artifacts must be identifiable, authentic, and non-repudiable
-1. Such Statements must be registered on a secure Append-only Log, enabling provenance and history to be independently and consistently audited
-1. Issuers can efficiently prove to any other party the Registration of their Signed Statements; verifying this proof ensures that the Issuer is consistent and non-equivocal when producing Signed Statements
+1. Statements made by Issuers about supply chain Artifacts must be identifiable, authentic, and non-repudiable.
```
Comment (SteveLasker) [inline draft-ietf-scitt-architecture.md:140]: Throughout the doc, we removed punctuation on single-sentence lines. If we want change that, we should do a single sweep for consistency

**Comment by SteveLasker** [comment]
This makes changes inconsistent with style throughout the doc. Suggest we either fix all references or close this PR to maintain consistency.

File: draft-ietf-scitt-architecture.md (hunk 1 above)
Comment (henkbirkholz) [reply draft-ietf-scitt-architecture.md:126]: Exactly.
But a) this this the abstract and b) how do we deal with usage of capitalized terms in the text before they are introduced in the terminology Section?

File: draft-ietf-scitt-architecture.md (hunk 2 above)
Comment (henkbirkholz) [reply draft-ietf-scitt-architecture.md:131]: see https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/157#discussion_r1457420130

File: draft-ietf-scitt-architecture.md (hunk 3 above)
Comment (henkbirkholz) [reply draft-ietf-scitt-architecture.md:140]: > Throughout the doc, we removed punctuation on single-sentence lines.
I am relatively certain this is not true :sweat_smile: Two examples to counter: line 201 (was 203) and line 386 (was 383).
What I did here was just to add periods to complete sentences that appear in itemized lists.

**Comment by henkbirkholz** [comment]
This PR is intended to discuss style proposals directly on text in order to derive issues from the discussion here.

**Pull request closed without merging**